understanding/20/three-flashes-or-below-threshold.html

```diff
@@ -38,12 +38,15 @@ <h2>Intent of Three Flashes or Below Threshold</h2>
          did not allow any flashing (even of a single pixel) within a broad frequency range
          (3 to 50 Hz). This Success Criterion is based on existing specifications in use in
          the UK and by others for television broadcast and has been adapted for computer display
-         viewing. The 1024 x 768 screen is used as the reference screen resolution for the
+         viewing. In WCAG 2.0, the 1024 x 768 screen was used as the reference screen resolution for the
          evaluation. The 341 x 256 pixel block represents a 10 degree viewport at a typical
          viewing distance. (The 10 degree field is taken from the original specifications and
          represents the central vision portion of the eye, where people are most susceptible
          to photo stimuli.)
       </p>
+      <p>With the proliferation of devices of varying screen sizes (from small hand-helds to large living room displays), as well as the adoption of <a href="https://www.w3.org/TR/WCAG21/#dfn-css-pixels">CSS pixels</a> as a density-independent unit of measurement, the prior assessment criteria seem outdated. However, an image of a consistent size uses up relatively the same percentage of a user's visual field on any device. On a large screen, the image takes up less size, but the large screen takes up a larger part of the visual field. On a mobile screen, the image may take up most or all of the screen; however, the mobile screen itself takes up a smaller portion of the user's visual field. So the same dimension of the flashing content, represented in CSS pixels can still provide a consistent means of assessment. Substituting CSS pixels for the original pixel block means that the assessment becomes 341 x 256 CSS pixels, or a flashing area of 87,296 CSS pixels.</p>
```

Comment:
`the prior assessment criteria seem outdated` reads a bit weird, especially since the next sentence clarifies why they're not. maybe `the prior assessment criteria *may* seem outdated`

Reply:
`the assessment becomes 341 x 256 CSS pixels` reads slightly off. maybe more accurately `the threshold for the combined area of flashing becomes ...`